### PR TITLE
Update git-fastclone.rb

### PR DESCRIPTION
**Pull request by gyfelton**

### CODE REVIEW

File: lib/git-fastclone.rb [hunk 1]
```diff
@@ -288,6 +289,7 @@ def with_reference_repo_lock(url, &block)
     ensure
       # Not strictly necessary to do this unlock as an ensure. If ever exception is caught outside this
       # primitive, ensure protection may come in handy.
+      puts "Waiting for another fastclone process to finish"
```
Comment (justinseanmartin) [inline lib/git-fastclone.rb:292]: I'm not sure this message is accurate. Aren't we yielding the lock, so other pending fastclone processes can start? What is the behavior you're encountering and trying to understand better?

File: lib/git-fastclone.rb [hunk 2]
```diff
@@ -217,7 +217,8 @@ def clone(url, rev, src_dir, config)
       with_git_mirror(url) do |mirror, attempt_number|
         clear_clone_dest_if_needed(attempt_number, clone_dest)
 
-        clone_command = '--quiet --reference :mirror :url :path'
+        clone_command = '--reference :mirror :url :path'
+        clone_command += verbose ? '--verbose' : '--quiet'
```
Comment (justinseanmartin) [inline lib/git-fastclone.rb:221]: Is `--verbose` passed to fastclone from anywhere that we know of today? Are we planning on adding it somewhere or using it as a one-off?

**Review by justinseanmartin — COMMENTED** [review]
I'm curious about the context for this change, would love to know more.

**Comment by gyfelton** [comment]
@justinseanmartin thx for having interest on this PR :) The problem I am having is that for some reason fast clone process in EC2 validation job seems to basically stuck without any message, example: https://mobuild.sqprod.co/blue/organizations/jenkins/CI%2Bios%2B3NTJQL%2BBazel-EC2-Validation/detail/CI%2Bios%2B3NTJQL%2BBazel-EC2-Validation/55/pipeline

I have not seriously look into this yet. Current theory is only new EC2 instance suffer this which matches with the actual (subsequent jobs all running fast) but this is def. needs to be fixed for new instances we just got (as then this seems random which is not)

**Pull request closed without merging**